develop-guide/writeblitz/writeblitz.rst

```diff
@@ -627,6 +681,12 @@ in action ``execute`` output and print the   :monospace:`main_learn_output` into
     Both filter and include/exclude features are using our dictionary querying tool `Dq
     <https://pubhub.devnetcloud.com/media/genie-docs/docs/userguide/utils/index.html#dq>`_.
 
+.. note::
+
+    The name of the device that the action is being executed on will be saved automatically upon
+    execution of the action and stay usable till the end of that action lifecycle. You can use that 
+    name as a variable using ``VARIABLES{device.name}`` for various purposes in your action. 
```

Comment:
missing `%`? `%VARIABLES{device.name}`. I think more friendly if we have an example of yaml.

Reply:
Done